src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  padding-right: 40px;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: row;
+  align-items: center;
+  height: 60px;
+  background-color: white;
+}
+
+.nav__link {
+  line-height: 60px;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  margin: 0 10px;
+}
+
+.logo {
+  z-index: 1;
+  margin: 10px 50px;
+}
+
+.is-active {
+  position: relative;
+  --is-active-width: 37px;
```

Comment:
This should not be fixed value. Use `left: 0` and `right: 0` for absolute positioning, or at least `width: 100%`.